can.js

```diff
@@ -27,6 +27,34 @@ export async function storeAdd (carPath) {
   spinner.stopAndPersist({ symbol: '⁂', text: `Stored ${cid}` })
 }
 
+/**
+ * Print out all the CARs in the current space.
+ * @param {object} opts
+ * @param {boolean} [opts.raw]
+ * @param {boolean} [opts.json]
+ * @param {string} [opts.cursor]
+ * @param {number} [opts.size]
+ * @param {boolean} [opts.pre]
+ */
+export async function storeList (opts = {}) {
+  const client = await getClient()
+  const listOptions = {}
+  if (opts.size) {
+    listOptions.size = parseInt(opts.size)
+  }
+  if (opts.cursor) {
+    listOptions.cursor = opts.cursor
+  }
+  if (opts.pre) {
+    listOptions.pre = opts.pre
+  }
+
+  const spinner = ora('Listing CARs').start()
```

Comment:
when `--json` is passed, the output should only be machine readable json or ndjson for robots and script. Emitting a spinner would make `jq` unhappy.

Reply:
ah so I was worried about this but I was careful to make sure `ora` plays nice and deletes this from the output that gets forwarded on to other tools! check it out:

https://user-images.githubusercontent.com/1113/218740661-c4698782-a451-470f-95e4-ff5260c4c54c.mov